### PR TITLE
added support for genes for good

### PR DESCRIPTION
**Pull request by gedankenstuecke**
This basically takes care of #290

### CODE REVIEW

File: app/views/static/_faq.html.erb [hunk 1]
```diff
@@ -246,7 +254,11 @@
...
+=======
+	          <strong>1.</strong>You should read the description on the phenotype you are about to enter. Often the description gives further details on which kind of answers people are interested in. You can also take a look at the answers which are given so far by other users. This should also give you a feeling about what kind of information is wanted for this phenotype. <br/><strong>2.</strong> As soon as you start to enter your own variation, we will try to auto-complete your writing to avoid duplicate entries into the openSNP database. Having multiple entries for the same thing, like "blue-green" plus "green-blue" in case of eye-colour makes it harder for (citizen) scientists to use the data for their studies. <br/><strong>3.</strong> The thing you are about to enter doesn't get auto-completed? Great. You are about to enter a variation which has not been reported so far, have fun entering your data!
+>>>>>>> master
```
Comment (philippbayer) [inline app/views/static/_faq.html.erb]: These look like forgotten overhead from conflict merging?

Comment (gedankenstuecke) [inline app/views/static/_faq.html.erb]: right, removed in the next commit!

**Comment by gedankenstuecke** [comment]
for now I'll ignore the other hound comments. Seems the complete preparsing job violates the hounds wishes for now 😉

**Comment by philippbayer** [comment]
I think Hound just complains about the parts that you've changed, it keeps quiet about lines you didn't touch.

Since the new test (nice) passes I'll merge it

**Comment by gedankenstuecke** [comment]
Thanks! Getting the test right was the hardest part! 😄

**Pull request merged**